aws/infrastructure/backend.tf

```
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    bucket  = "terraform-state-xain"
+    key     = "infrastructure/terraform.tfstate"
```

If we decide to use the other file layout, tthat would need to be changed to:
```suggestion
    key     = "global/s3/terraform.tfstate"
```